### PR TITLE
[Icons] Added more UI icons

### PR DESCRIPTION
**Pull request by pkwi**
Added more UI icons: Tick circle, Prescription and New prescription.

### CODE REVIEW

File: packages/core/resources/icons/icons.json [hunk 1]
```diff
@@ -2840,5 +2840,26 @@
     "tags": "sidebar, layout, spreadsheet",
     "group": "table",
     "content": "\\e778"
+  },
+  {
+    "name": "Tick circle",
+    "className": "pt-icon-tick-circle",
+    "tags": "tick mark, sign, ok, approved, success",
```
Comment (llorca) [inline packages/core/resources/icons/icons.json]: @pkwi Shouldn't `tick mark` be `tick, mark` instead?

Comment (giladgray) [inline packages/core/resources/icons/icons.json]: doesn't even need to contain "tick," cuz that's part of the name. "mark" should be enough

File: packages/core/resources/icons/icons.json [hunk 2]
```diff
@@ -2840,5 +2840,26 @@
...
+    "group": "action",
+    "content": "\\e779"
+  },
+  {
+    "name": "Prescription",
```
Comment (giladgray) [inline packages/core/resources/icons/icons.json]: this doesn't feel like a useful UI icon, it's specific to certain use cases. why do we need?

Comment (llorca) [inline packages/core/resources/icons/icons.json]: Agree, though there already are a bunch of specific icons that got in previously (i.e. rig, field, known vehicle, blocked person...). We could clean it up in 2.0 by keep specific ones internally?

Comment (giladgray) [inline packages/core/resources/icons/icons.json]: ok fair

**Pull request merged**